src/main/java/com/playmonumenta/networkrelay/RemotePlayerManager.java

```diff
@@ -45,25 +55,27 @@ public RemotePlayer(JsonObject remoteData) {
 			mIsHidden = remoteData.get("isHidden").getAsBoolean();
 			mIsOnline = remoteData.get("isOnline").getAsBoolean();
 			mShard = remoteData.get("shard").getAsString();
-			mPluginData = serializePluginData(remoteData);
+			mPluginData = deserializePluginData(remoteData);
+
+			MMLog.fine("Received RemotePlayerState for " + mName + " from " + mShard + ": " + (mIsOnline ? "online" : "offline"));
 		}
 
 		@Nullable
 		public JsonObject getPluginData(String pluginId) {
 			return mPluginData.get(pluginId);
 		}
 
-		private ConcurrentMap<String, JsonObject> serializePluginData(JsonObject remoteData) {
+		private ConcurrentMap<String, JsonObject> deserializePluginData(JsonObject remoteData) {
```

Comment:
Serialize -> Convert to string (or json in this case)
Deserialize -> Convert from string

Reply:
I knew I would make that mistake at least once in my life